File: server/medicore.layer/MediCore.Service/Model/UTCTimeBaseModel.cs
using dna.core.data.Infrastructure;
using dna.core.service.Models.Abstract;
using System.ComponentModel.DataAnnotations;

namespace MediCore.Service.Model
{
    public class UTCTimeBaseModel : IModelBase
    {
        public int Id { get; set; }
        [Required(ErrorMessage = "Required")]
        public int CountryId { get; set; }

        [Required(ErrorMessage = "Required")]
        [StringLength(100, ErrorMessage = "StringLength")]
        public string Name { get; set; }

        [Required(ErrorMessage = "Required")]
        [StringLength(10, ErrorMessage = "StringLength")]
        public string Code { get; set; }

        //UTC is +8, -8, etc
        [Required(ErrorMessage = "Required")]
        public int UTC { get; set; }

        [Required]
        public Status Status { get; set; }

        public virtual CountryModel Country { get; set; }
    }
}

File: smartmedika.admin/src/app/model/utcTimeBaseModel.ts
import { IEntityBase }          from './IEntityBase';
import { CountryModel }         from './countryModel';
import { Status } from './enumModel';

export class UTCTimeBaseModel implements IEntityBase{
    id: number;
    countryId: number;
    name: string;
    code: string;
    utc: number;
    status: Status;
    country: CountryModel;

    constructor(){
        this.id = 0;
        this.countryId = 0;
        this.name = null;
        this.utc = 0;
        this.country = null;
    }
}